src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -600,20 +593,22 @@ public static class TextCatalog
         public static LatentDirichletAllocationEstimator LatentDirichletAllocation(this TransformsCatalog.TextTransforms catalog,
```

Comment:
>LatentDirichletAllocation [](start = 57, length = 25)

not directly related to your change, but the other names are usually verbs. If we wanted to keep to that, shall this be named ApplyLatentDirichletAllocation

Reply:
Not sure. `ConvertLatentDirichletAllocation` looks a bit confusing. For example, we don't say `ConvertToMatrixFactorization`. One proper (mathematically) choice is `ProduceConvertLatentDirichletAllocationVector` but it's just too long..

---
In reply to: [264332826](https://github.com/dotnet/machinelearning/pull/2890#discussion_r264332826) [](ancestors = 264332826)